azure-devops/cross-build.yml

```diff
@@ -9,23 +9,14 @@ parameters:
   type: string
 - name: vsDevCmdArch
   type: string
-- name: buildOutputLocationVar
-  type: string
-  default: buildOutputLocation
-- name: benchmarkBuildOutputLocationVar
-  type: string
-  default: benchmarkBuildOutputLocation
+- name: buildBenchmarks
+  type: boolean
+  default: true
 - name: numShards
   type: number
   default: 8
 jobs:
 - job: '${{ parameters.targetPlatform }}'
-  variables:
-    fixedFlags: '--timeout=240;--shuffle'

```

Comment:
I forgot to mention in the commit log that I removed this `--shuffle` flag that is overridden by the later `--order=lexical`.

Reply:
Ooh, good cleanup, this was potentially hazardous if never overridden.

Reply:
Updated the PR description to mention this. Also mentioned the improved path quoting that I noticed elsewhere.

Reply:
> Ooh, good cleanup, this was potentially hazardous if never overridden.

Yes, I don't know what we were thinking in https://github.com/microsoft/STL/pull/2795.